API_exercise/api_exercise.py

```diff
@@ -0,0 +1,20 @@
+from flask import Flask, request, jsonify
+from zadnie_csv import convert_to_json
+app = Flask(__name__)
+app.debug = True
+
+
+@app.route('/process_csv', methods = ['POST'])
+def process_csv():
+	data = convert_to_json()
```

Comment:
W tym miejscu tworzysz 'data'  z pliku. 
A w API chodzi o to, że przychodzi Ci request z zewnątrz a dane są w Body tego requestu. 
Czyli trzeba zbadać obiekt: request.data
https://stackoverflow.com/questions/10434599/get-the-data-received-in-a-flask-request